doc/jsonrpc-api.md

```diff
@@ -325,3 +325,71 @@ Response (values omitted for brevity):
     }
 }
 ```
+
+### invokeContract
+
+Simulates a smart contract update, and returns information about the specified smart contract instance.
+
+#### Parameters
+- `blockHash` - hex encoding of a block's hash
+- `context` - a collection of parameters used to invoke a contract
+- `context.contract` - contains the index and subindex of the smart contract instance that should be invoked
```

Comment:
To remain consistent you should have:
```
context.contract.index
context.contract.subindex
```
instead to be explicit about the naming of the parameters.